app/test_main.py

```diff
@@ -1,3 +1,17 @@
+import pytest
+
```

Comment:
good code, but you dont need an empty line when you import modules

Reply:
This is true only for modules of the same type. Between third-party imports and local application specific imports, you should have an empty line.